README.md

```diff
@@ -136,6 +136,8 @@ and for Android:
 
 `npm run test:e2e:android:local`
 
+**Note:** Make sure you've run the above commands at least once, so the demo app binaries are built before running individual tests below.
+
```

Comment:
Alternatively, we could provide the commands to generate the demo app binaries:
- Android: `npm run test:e2e:bundle:android && npm run core test:e2e:build-app:android`
- iOS: `npm run test:e2e:bundle:ios && npm run core test:e2e:build-app:ios`

Reply:
Thought about that but felt it's becoming a bit verbose since we need to mention both platforms and the commands are currently lengthy. And to be honest, I feel that the build scripts should be run automatically before the test run (but I think we were not doing that originally because it takes time).

No strong feelings on this one though, perhaps it's still best to add the commands to the note, wdyt?

Reply:
No strong feelings either from my side, and I agree about making it too verbose. Let's stick with the note without the commands.